README.md

```diff
@@ -164,4 +186,4 @@ explain why.
 
 ## A2.7
 
-«replace this with your answer»
+This would be a type 2 grammar
```

Comment:
# -3

It's a type 3, because it you've shown it can be implemented by a state machine.